### ISSUE TITLE
Train my own dataset

### ISSUE THREAD

**Issue by InstantWindy** [idealo/image-super-resolution#132]
Hi. I use this repro to train my own dataset. During the training process, the PSNR value of my test set was decreasing. What is the reason for this？
 my low resolution dataset (down x2) is to scale the high-resolution dataset to half its original size. That is to say, my low resolution data sets are collected from high-resolution data sets. Thanks.

**Comment by cfrancesco** [comment]
Hard to say, how many data points are you using for training and test? Is PSNR increasing for training set? What config are you using?

**Comment by svenrog** [comment]
I had similar issues when trying to train a quite large set of product shots with a lot of white space around them.
Since the discriminator has a fixed flatness set, that used patches randomly containing no detail throughout the training process and when the flatness schedule of the model increased comparisons started to differ.

To remedy that problem I set the flatness increase parameter to 0.
```
flatness = {'min': 0.00, 'max': 0.15, 'increase': 0.0, 'increase_frequency': 5 }
```

I also think I encountered an issue with the adam optimizer used in the trainer.
There is a comment inside `trainer.py` linking to this article https://stackoverflow.com/questions/42327543/adam-optimizer-goes-haywire-after-200k-batches-training-loss-grows

I set the epsilon value to 0.1 like below.
```
optimizer = {'beta1': 0.9, 'beta2': 0.999, 'epsilon': 0.1 }
```

Training went fine after that. Hope it helps someone.
